src/lib/libhtml5.js

```diff
@@ -201,6 +201,25 @@ var LibraryHTML5 = {
       return {{{ cDefs.EMSCRIPTEN_RESULT_SUCCESS }}};
     },
 
+    removeSingleHandler(eventHandler) {
+      if (!eventHandler.target) {
+#if ASSERTIONS
+        err('removeSingleHandler: the target element for event handler registration does not exist, when processing the following event handler registration:');
+        console.dir(eventHandler);
+#endif
+        return {{{ cDefs.EMSCRIPTEN_RESULT_UNKNOWN_TARGET }}};
+      }
+      for (var i = 0; i < JSEvents.eventHandlers.length; ++i) {
+        if (JSEvents.eventHandlers[i].target === eventHandler.target
+          && JSEvents.eventHandlers[i].eventTypeId === eventHandler.eventTypeId
+          && JSEvents.eventHandlers[i].callbackfunc === eventHandler.callbackfunc
+          && JSEvents.eventHandlers[i].userData === eventHandler.userData) {
+          JSEvents._removeHandler(i--);
```

Comment:
Can you just `return {{{ cDefs.EMSCRIPTEN_RESULT_SUCCESS }}};` here?

And should we `return {{{ cDefs.EMSCRIPTEN_RESULT_INVALID_PARAM }}};`  below if its not found?